client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+const Suggestions = require( './suggestions-mixin' );
+
+export default React.createClass( {
+	displayName: 'Mentions',
+
+	mixins: [ Suggestions ],
+
+	getInitialState: function() {
+		return {
+			suggestionsVisible: false
+		};
+	},
+
+	componentDidMount: function() {
+		const self = this,
+			editor = this.props.editor;
+
+		function onEditorKeyUp( event ) {
```

Comment:
What about using the [arrow function](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) pattern here? We could avoid the `const self = this` that way because arrow functions have lexical `this`.

For example:

```js
const onEditorKeyUp = ( event ) => {
    this.handleSuggestionsKeyUp( event, editor.getBody(), editor.getContent( { format: 'text' } ) );
};
```